src/lib/storage/table.hpp

```diff
@@ -85,6 +85,7 @@ class Table : private Noncopyable {
   // returns the chunk with the given id
   std::shared_ptr<Chunk> get_chunk(ChunkID chunk_id);
   std::shared_ptr<const Chunk> get_chunk(ChunkID chunk_id) const;
+  void remove_chunk(ChunkID chunk_id);
```

Comment:
This function decl seems a bit squeezed in here... Can you give it its own block and document it?